packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -290,8 +296,76 @@ $search-transition-timing: 95ms;
     &:active {
       border-color: $focus;
       background-color: $hover-ui;
+      display: block;
+      &::before,
+      &::after {
+        display: none;
+      }
+    }
+    &-left-caret,
```

Comment:
We are going away from BEM concatenation (`&-foo`) for better reuse of style rules between React and Web Components.

Reply:
fixed! Thanks @asudoh!